aikau/src/main/resources/alfresco/renderers/InlineEditProperty.js

```diff
@@ -369,19 +365,27 @@ define(["dojo/_base/declare",
        * @returns {object} The widget for editing.
        */
       getFormWidget: function alfresco_renderers_InlineEditProperty__getFormWidget() {
-         if (this.formWidget === null)
+         if (!this.formWidget)
          {
             var uuid = this.generateUuid();
-            this.alfSubscribe(uuid + "ALF_FORM_VALIDITY", lang.hitch(this, this.onFormValidityChange), true);
+            var saveTopic = "_SAVE";
+            var cancelTopic = "_CANCEL";
+            this.alfSubscribe(uuid + saveTopic, lang.hitch(this, this.onSave), true);
```

Comment:
Would be good refactor this at some point to allow specification of a scope in the arguments, rather than having to do string concatenation. Perhaps a candidate for 1.1.0?